pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -236,6 +238,23 @@ func (opt *Options) Run(args []string) error {
 	// anomalies
 	var syntheticTestResults []*JUnitTestCase
 	if events := m.Events(time.Time{}, time.Time{}); len(events) > 0 {
+		// Serialize the interval data for easier external analysis
+		intervalsFilename := filepath.Join(opt.JUnitDir, "intervals.json.gz")
+		if file, err := os.Create(intervalsFilename); err != nil {
+			fmt.Fprintf(opt.Out, "ERROR: failed to create %s: %v\n", intervalsFilename, err)
```

Comment:
Error messages go to `opt.ErrOut`, and your prefix should be `error: Failed to create event output: %v` (don't display the filename, display what you're trying to do because the os error usually has the filename).

Reply:
thanks, fixed